src/index.html

```diff
@@ -5,8 +5,293 @@
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#apple" class="header__link is-active">APPLE</a>
+          </li>
+          <li class="header__item">
+            <a href="#samsung" class="header__link">SAMSUNG</a>
+          </li>
+          <li class="header__item">
+            <a href="#смартфоны_и_телефоны" class="header__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#ноутбуки_и_компьютеры"
+              class="header__link"
+              data-qa="nav-hover"
+            >НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="header__item">
+            <a href="#гаджеты" class="header__link">ГАДЖЕТЫ</a>
+          </li>
+          <li class="header__item">
+            <a href="#планшеты" class="header__link">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="header__item">
+            <a href="#фото" class="header__link">ФОТО</a>
+          </li>
+          <li class="header__item">
+            <a href="#видео" class="header__link">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog">
+        <div class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="imac"
+          >
+          <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+          <p class="card__code">Код товара: 195434</p>
+          <div class="card__feedback">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <span class="card__review">Отзывов: 5</span>
+          </div>
+          <div class="card__price">
+            <span class="card__price-name">Цена:</span>
+            <span class="card__price-amount">69 999 грн</span>
+          </div>
+          <a
+            href="#button"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            КУПИТЬ
+          </a>
+        </div>
+        <div class="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="imac"
+          >
+          <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+          <p class="card__code">Код товара: 195434</p>
+          <div class="card__feedback">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
```

Comment:
Why stars don't have a modifier? The number of yellow stars must be specified by the modifier. 
3rd item in the [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) to the product card